src/App.tsx

```diff
@@ -17,35 +20,40 @@ export const App = () => (
     >
       <div className="container">
         <div className="navbar-brand">
-          <a href="/" className="navbar-item is-active">Home</a>
-          <a href="/tabs" className="navbar-item">Tabs</a>
+          <NavigationLink
+            link="/"
+            text="Home"
+          />
+          <NavigationLink
+            link="/tabs"
+            text="Tabs"
+          />
```

Comment:
If you have < 3 attribues on a tag and the values are short write the tag in one line (to make it easier to write and read).
Check through all code